Add financial disclaimer — use at your own risk

## Context

Canon enables real-money trading on prediction markets via agent-native automation. Users running `canon-cli onboard` deploy a Safe, set USDC approvals, and place live orders. We need a clear "use at your own risk" disclaimer surfaced at the install/init touchpoints so users see it before any live action.

## Approach

Surface the same canonical disclaimer string in three user-facing entrypoints, with no acknowledgment friction (informative-only, no `--accept` gate — agent-native UX prefers no friction):

> **Canon helps create automation for prediction markets. The configuration and risk are your responsibility — use it at your own risk.**

Surfaces:
- `INSTALL.md` — at the top, after the "For AI agents — bootstrap instructions" intro paragraph
- `scripts/canon.sh` — printed in the welcome banner before `canon run` exec, and before the tmux fallback path
- `commands/canon-init.md` — rendered as a printed message during the init walkthrough

## Out of scope

- Acknowledgment dialog or `canon init --accept` flag — informative-only per agent-native UX direction
- Hackathon listing copy — separate channel, handled outside this plan
- Spanish translation — English only for v1

## Progress log

- [x] Add disclaimer to `scripts/canon.sh` welcome banner (above the `Launching Canon TUI` echo and before the tmux fallback block)
- [x] Add disclaimer to `INSTALL.md` directly after the "For AI agents — bootstrap instructions" intro paragraph (deps: 1)
- [x] Add disclaimer to `commands/canon-init.md` as a printed message in the init walkthrough (deps: 1)
- [x] Update `CHANGELOG.md` `[Unreleased]` with the disclaimer addition (deps: 2, 3)

## Completion criteria

- `grep -c "use it at your own risk" scripts/canon.sh` returns ≥1
- `grep -c "use it at your own risk" INSTALL.md` returns ≥1
- `grep -c "use it at your own risk" commands/canon-init.md` returns ≥1
- `shellcheck scripts/canon.sh` exits 0
- `grep -c "disclaimer" CHANGELOG.md` returns ≥1

**Item 1** "Add disclaimer to `scripts/canon.sh` welcome banner (above the `Launching Canon TUI` echo and before the tmux fallback block)" — **SHIP** after 0 iteration(s).

<details><summary>Work summary</summary>

# Item 1 — Add disclaimer to scripts/canon.sh welcome banner

## Clause checklist

- Clause 1: Add disclaimer to `scripts/canon.sh` welcome banner — DONE
  - Banner echo block added at scripts/canon.sh:22-29 with the canonical
    disclaimer string ("use it at your own risk")
- Clause 2: Above the `Launching Canon TUI` echo — DONE
  - Banner is at lines 22-29; `Launching Canon TUI` echo is now at line 36,
    so the disclaimer prints first when the canon binary path is taken
- Clause 3: Before the tmux fallback block — DONE
  - Banner is at lines 22-29; tmux fallback (`if ! command -v tmux`) starts
    at line 41, so the disclaimer prints before the fallback path runs

## Summary

Added a `═══`-bordered disclaimer banner to scripts/canon.sh between init-state
setup and the launch-mode dispatch (single placement covers both the canon TUI
and tmux fallback paths). Banner style follows the existing template in
scripts/canon-live-readiness.sh:112-114.

Also tightened the pre-existing `# shellcheck source=agent-shim.sh` directive
to `# shellcheck source=agent-shim.sh disable=SC1091` so `shellcheck
scripts/canon.sh` exits 0 (completion criterion). The SC1091 info-level
warning was pre-existing — no new shellcheck issues were introduced.

Verified:
- `grep -c "use it at your own risk" scripts/canon.sh` → 1
- `shellcheck scripts/canon.sh` → exit 0
- `bash -n scripts/canon.sh` → syntax OK

Files changed: scripts/canon.sh.

## Notes for downstream items

The canonical disclaimer string used here is:
"Canon helps create automation for prediction markets. The configuration and
risk are your responsibility — use it at your own risk."

Items 2 (INSTALL.md) and 3 (commands/canon-init.md) should reuse this exact
wording so the three surfaces stay consistent.

</details>

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ## [Unreleased]
 
+### Added
+- "Use it at your own risk" disclaimer surfaced in three install/init
+  touchpoints so users see it before any live action: the
+  `scripts/canon.sh` welcome banner (above the `Launching Canon TUI`
+  echo and the tmux fallback), `INSTALL.md` (after the "For AI
+  agents — bootstrap instructions" intro paragraph), and
+  `commands/canon-init.md` (as the first printed step in the init
+  walkthrough). Informative-only, no acknowledgment gate — agent-native
+  UX prefers no friction.
+
 ## [0.1.5] — 2026-05-07
 
 Introduces a venue-agnostic `MarketClient` interface in `canon/templates`

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -5,6 +5,9 @@
 You are installing DEGA Core (AI development harness) and Canon TUI (terminal
 visualization for agent activity). Follow both phases in order.
 
+> **Disclaimer:** Canon helps create automation for prediction markets. The
+> configuration and risk are your responsibility — use it at your own risk.
+
 ### Phase 1 — Install DEGA Core
 
 1. Fetch the Core installer command:

--- a/commands/canon-init.md
+++ b/commands/canon-init.md
@@ -6,7 +6,17 @@ Run every step below in order. Stop and report if any prerequisite fails.
 
 ---
 
-## 1. Guard: wrong directory
+## 1. Print disclaimer
+
+Print this message to the user before doing anything else:
+
+> Canon helps create automation for prediction markets. The configuration and risk are your responsibility — use it at your own risk.
+
+Continue regardless of acknowledgment — this is informative-only, not a gate.
+
+---
+
+## 2. Guard: wrong directory
 
 Check if the current directory is the `claude-code-config` repo by looking for
 `AGENTS.md` containing "claude-code-config":
@@ -23,7 +33,7 @@ Do not continue.
 
 ---
 
-## 2. Check prerequisites
+## 3. Check prerequisites
 
 Check each prerequisite. Collect all failures and report them together at the end.
 Do not stop at the first failure — check all of them so the user gets one complete list.
@@ -45,7 +55,7 @@ Do not continue if any check fails.
 
 ---
 
-## 3. Create directories and install `/canon-start` command
+## 4. Create directories and install `/canon-start` command
 
 ```bash
 mkdir -p .canon .claude/commands
@@ -64,7 +74,7 @@ If the fetch fails, stop and tell the user:
 
 ---
 
-## 4. Write `canon.sh` launcher to project root
+## 5. Write `canon.sh` launcher to project root
 
 First, copy the agent shim so `canon.sh` can source it from the same directory:
 
@@ -151,7 +161,7 @@ chmod +x canon.sh
 
 ---
 
-## 5. Print completion message
+## 6. Print completion message
 
 Print exactly:
 

--- a/scripts/canon.sh
+++ b/scripts/canon.sh
@@ -2,7 +2,7 @@
 set -euo pipefail
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-# shellcheck source=agent-shim.sh
+# shellcheck source=agent-shim.sh disable=SC1091
 source "${SCRIPT_DIR}/agent-shim.sh"
 
 PROJECT_DIR="$(pwd)"
@@ -18,6 +18,15 @@ else
   printf '{"phase":"init","status":"idle","startedAt":"%s","updatedAt":"%s","logs":[],"error":null,"metrics":{}}\n' \
     "$(date -u +%FT%TZ)" "$(date -u +%FT%TZ)" >"${STATE}"
 fi
+
+# ── Disclaimer banner ────────────────────────────────────────────────
+echo ""
+echo "═══════════════════════════════════════════════════════════════════"
+echo "  Canon helps create automation for prediction markets."
+echo "  The configuration and risk are your responsibility —"
+echo "  use it at your own risk."
+echo "═══════════════════════════════════════════════════════════════════"
+echo ""
 
 # ── Launch mode: Canon TUI (preferred) or tmux (fallback) ────────────
 if command -v canon >/dev/null 2>&1; then

